open_lm/main.py

```diff
@@ -118,9 +129,10 @@ def load_model(args, model):
     else:
         # loading a bare (model only) checkpoint for fine-tune or evaluation
         start_epoch, global_step = 0, 0
+        pretrained_seed = None
         model.load_state_dict(checkpoint)
         logging.info(f"=> loaded checkpoint '{args.resume}' (epoch {start_epoch})")
-    return start_epoch, global_step
+    return start_epoch, global_step, pretrained_seed
```

Comment:
why do we have to return `pretrained_seed` isnt it just in `args`?

Reply:
This is done for backwards compatibility with runs that did not have a pretrained seed - in that case, I set `pretrained_seed = None` so that no shuffling is performed (and the original behavior is preserved).